src/javaRosa.js

```diff
@@ -1388,6 +1388,9 @@ define([
                     insertOutputRef(_this, target, path, mug);
                     if (window.analytics) {
                         window.analytics.usage("Output Value", "Drag and Drop");
+                        if (path.contains('casedb')) {
+                            window.analytics.usage("Case Management", "Form Builder", "Click and Drag");
```

Comment:
Two different phrases are used to mean basically the same thing ("Drag and Drop" vs "Click and Drag"). Not sure if this is by design or if it could create two categories where one is preferred. Also, the arguments for the `usage` function are `category, action, label, value`. If I'm not mistaken action is "Drag and Drop" on line 1390 and "Form Builder" on line 1392—is that intended? @amsagoff may have input as well.